specification/web/resource-manager/Microsoft.CertificateRegistration/stable/2022-09-01/AppServiceCertificateOrders.json

```diff
@@ -1035,6 +1035,53 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CertificateRegistration/certificateOrders/{name}/retrieveCertificateOrderContact": {
```

Comment:
>subscriptions [](http://example.com/codeflow?start=6&length=13)

It is better to model these actions as proper REST objects and convert these to GET calls. Whats the reason you are implementing all of these as a bunch of POST actions?